src/twisted/internet/endpoints.py

```diff
@@ -1414,6 +1415,9 @@ def _parseSSL(
         constant in C{OpenSSL.SSL}.
     @type sslmethod: C{str}
 
+    @param cipher: A string containing a list of cipher values, formatted as comma-separated values.
```

Comment:
What is a "cipher value"? Presumably there is a list of these magic strings somewhere. Is it OpenSSL specific? How should the user discover the possible list of options? Why might they want to use this option?

Reply:
Hi @glyph yes, so if you do openssl ciphers -v 'ALL:eNULL', you can find all the cipher values 

The usecase is, I work as a developer in a bank and they want to limit cipher use, even in so basic requirement is TLS1.2 and even in there, they only want to enable a few ciphers

They are already using Daphne and Daphne is using `serverFromString` from Twisted 

They are averse of making big changes like using Twisted directly, so with this I can have them use only selected cipher for them

I was going to make this changes directly in Daphne but this could be used by a lot of people 
So doing it here made more sense to me

Reply:
Should i mention from where to get the -cipher value from? 
I think people wanting to use this feature would actually know what they are looking for and will specifically try finding it, like I id haha

Also thanks for `Twisted` you guys <3

Reply:
> The usecase is, I work as a developer in a bank and they want to limit cipher use, even in so basic requirement is TLS1.2 and even in there, they only want to enable a few ciphers

Sorry, I should have been clearer: I am asking these questions not because I don't know the answer (I do) but from the perspective of some random future developer reading about this in Twisted.  Twisted right now uses OpenSSL for TLS, but hopefully we will not always do that (I have a pretty short-term to-do item to look at pyrtls to see if we can get a more memory-safe option).

So, imagine that you are reading this as someone who knows what TLS is, and wants to do something with it, but doesn't know anything in particular about OpenSSL or how to use it.

Reply:
So the questions are all things that need to be somehow answered within the text of the source code, not just in a comment reply :)

Reply:
@glyph added it, let me know if its good enough or do I need to add more :)